internal/policies/manager.go

```diff
@@ -127,6 +136,7 @@ func NewManager(bus *dbus.Conn, opts ...Option) (m *Manager, err error) {
 		cacheDir:    consts.DefaultCacheDir,
 		runDir:      consts.DefaultRunDir,
 		apparmorDir: consts.DefaultApparmorDir,
+		unitDir:     "/etc/systemd/system",
```

Comment:
I thought about adding it there... I just thought it was something specific to the mount manager and, since I made a similar mistake with the user mounts, I decided to leave it in the mount manager 😅.